### ISSUE TITLE
Add -recursive parameter

### ISSUE THREAD

**Issue by mihailotomic** [securego/gosec#591]
### Summary 
In order to recursively some directory user must append `./...` to target directory.   
It would be useful for automation if there's an optional `-recursive` parameter (like in bandit [example](https://github.com/PyCQA/bandit/blob/bcebfc8f5397b5f52d10a66f9bf3c894e5060d6a/bandit/cli/main.py#L148)) that could automatically append target dir with `./...`.

**Comment by mihailotomic** [comment]
I agree it's natural for Go developers and removing `./...` wouldn't make any sense ofc.
But from a cybersecurity aspect, most of the open-source tools are _parameter driven_. If it's not much of the work, this feature could bring benefits to more security-oriented users of this great tool. 

Anyway, thank you for the quick response!

**Comment by ccojocar** [comment]
I leave it open such that we can have something for this. I am happy to accept a pull request if you want to implement it.

**Comment by ArnPellesGit** [comment]
Did you solve this? I wrote a bit and I think I came up with a solution. Only been modifying the cmd/gosec/main.go file though. Am I perhaps missing something obvious here? Tests are passing :)

**Comment by ccojocar** [comment]
No, I am happy to accept a pull request.

**Comment by ArnPellesGit** [comment]
I just created a PR. I'm fairly new to contributing to open source so I'd like to apologize in advance if I'm doing something wrong
